driver.go

```diff
@@ -106,7 +106,12 @@ type Driver struct {
 	logger hclog.Logger
 
 	// podmanClient encapsulates podman remote calls
-	podmanClient *PodmanClient
+	podmanClient2 *apiclient.APIClient
```

Comment:
since we are removing the varlink client can we just keep this as `podmanClient`